plugins/decklink/decklink-device.hpp

```diff
@@ -21,6 +21,7 @@ class DeckLinkDevice {
 	decklink_bool_t supportsInternalKeyer = false;
 	int64_t subDeviceIndex = 0;
 	int64_t numSubDevices = 0;
+	int64_t minimumPrerollFrames = 3;
```

Comment:
This seems redundant given the FAILED() check above. Otherwise, looks good to me

Reply:
Yeah, I thought about it, and I figured it was better to have a reasonable value here "just in case."